app/concepts/lockstep/api_record.rb

```diff
@@ -822,6 +815,17 @@ def destroy
       false
     end
 
+    def self.destroy_all(id: [])
+      id.each_slice(100) do |sliced_ids|
+        resp = resource.delete('', body: { 'idsToDelete' => sliced_ids })
+        return false unless resp.code.to_s == '200'
```

Comment:
bro what if one batch is deleted and other is not. should we collect the ids that are deleted and maybe return the count of deleted records, or ids of deleted records so that we can log what were deleted and what were not deleted? Kind of how rails does it with upsert_all?

Reply:
Added a TODO as discussed